templates/cppLibrary/android/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.4.1)
+
+set (CMAKE_VERBOSE_MAKEFILE ON)
+set (CMAKE_CXX_STANDARD 11)
+
+add_library(cpp
```

Comment:
Maybe choose another name for library. cpp maybe a little confusing.

Reply:
Actually it's a good idea. I will try to address that in next pr 👍